File: AndroidApp/MultiSportTrainer-Mobile/app/src/main/java/com/example/multisporttrainer/models/SaveResultRequest.java
package com.example.multisporttrainer.models;

public class SaveResultRequest {

    private int sessionId;
    private int userId;
    private int score;
    private double accuracy;
    private int mistakes;
    private int durationSeconds;

    public SaveResultRequest(
            int sessionId,
            int userId,
            int score,
            double accuracy,
            int mistakes,
            int durationSeconds
    ) {
        this.sessionId = sessionId;
        this.userId = userId;
        this.score = score;
        this.accuracy = accuracy;
        this.mistakes = mistakes;
        this.durationSeconds = durationSeconds;
    }
}

File: Backend/MultiSportTrainerAPI/MultiSportTrainerAPI/DTOs/SaveResultRequest.cs
namespace MultiSportTrainerAPI.DTOs
{
    public class SaveResultRequest
    {
        public int SessionId { get; set; }

        public int UserId { get; set; }

        public int Score { get; set; }

        public decimal Accuracy { get; set; }

        public int Mistakes { get; set; }

        public int DurationSeconds { get; set; }
    }
}